### PR TITLE
Support webhooks for esa.io (\( ⁰⊖⁰)/)

### PR DESCRIPTION
**Pull request by kakutani**
Now this supports following events in esa.io:

- post_create
- post_update
- post_archive
- comment_create
- member_join

_5__idobata

See also official document for details:
https://docs.esa.io/posts/37

To make sure `String#strip_heredoc` available in a hook preview env,
I would like to add require `active_support/core_ext/string/strip` in `lib/idobata/hook.rb`.

### CODE REVIEW

**Comment by aridori** [comment]
( ⁰⊖⁰) :+1:

File: lib/hooks/esa/templates/default.html.haml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+%p
+  Something happened on '#{payload.kind}' at
+  = link_to_post(payload.team, payload.post)
+  Chek it out (\( ⁰⊖⁰)/)
```
Comment (eitoball) [inline lib/hooks/esa/templates/default.html.haml]: `Chek` -> `Check`?

Comment (kakutani) [inline lib/hooks/esa/templates/default.html.haml]: oh...

**Comment by taea** [comment]
ﾍﾞﾝﾘ!! (\\( ⁰⊖⁰)/)

**Comment by hibariya** [comment]
Very nice (\\( ⁰⊖⁰)/) I'm trying this on test server. Please wait a minute :whale:

**Comment by hibariya** [comment]
This is now available on idobata.io (\\( ⁰⊖⁰)/) Thanks!

**Pull request merged**